### PR TITLE
add support for OpenSSL static

### PR DESCRIPTION
**Pull request by marler8997**

### CODE REVIEW

File: test.zig [hunk 1]
```diff
@@ -62,8 +65,8 @@ pub fn main() !u8 {
         try tests.append(Test.init(&[_][]const u8 { ziget, "google.com"}));
         try tests.append(Test.init(&[_][]const u8 { ziget, "http://google.com"}));
     }
-    if (openssl) {
```
Comment (g-w1) [inline test.zig:65]: Do you want to keep the regular openssl too? Or is it going away in favor of the static one?

Comment (marler8997) [reply test.zig:65]: Is there any reason to remove it?

Comment (g-w1) [reply test.zig:65]: No, but you removed it in the tests, that is why I am asking.

Comment (marler8997) [reply test.zig:65]: woops

**Pull request merged**